packages/bits/demo/src/components/demo/combobox-v2/combobox-v2-virtual-scroll/combobox-v2-virtual-scroll.example.component.html

```diff
@@ -1,4 +1,5 @@
 <nui-combobox-v2 placeholder="Select Item" i18n-placeholder
+                 [overlayConfig]="{ minHeight: 310 }"
```

Comment:
This doesn't seem quite right to have consumers arbitrarily set the overlay config min height to this magic number. Do you mind trying to preserve the calculateContainerHeight functionality and make it work to somehow avoid this?

Reply:
add getter and comment for this, thanks.

Reply:
@Blakesalot I'm not sure that root component (Select in our case) should look inside content projection. But, I recognize that my solution still doesn't work exactly right, so I provide another solution for it. Please let me know if new approach applicable for us.